cabra/src/pages/components/CardField.tsx

```diff
@@ -0,0 +1,35 @@
+import { ReactNode } from "react";
+import styled from "@emotion/styled";
+import tw from "twin.macro";
+
+type Span = 1 | 2 | 3;
+type Align = "left" | "center" | "right";
+
+const Title = styled.div`
+  ${tw`text-xs font-extrabold text-left`}
+`;
+
+const Field = styled.div<{ $span: Span; $align: Align }>`
+  ${tw`bg-secondary rounded-lg p-2`}
+  ${({ $span, $align }) => `
+    grid-column: span ${$span || 1} / span ${$span || 1};
+    text-align: ${$align};
+    `}
```

Comment:
Any reason the `$dollar` notation here? The `CardField` below doesn't have that. Is it some PHP flashback? xD

Reply:
There are some variables that are accessible using this notation (e.g. theme), so I usually use another notation here. Anyway, it should work if I remove the dollar signs

Reply:
In other words – HTML, theme and custom attributes are mixed here, so it's better to change the name